client/src/components/PrivateRoute.js

```diff
@@ -5,10 +5,10 @@ import { useAuth } from "../util/authContext";
 // authenticated. Takes the same props as a Route component from react-router.
 function PrivateRoute({ children, ...rest }) {
   const auth = useAuth();
-
+  console.log(auth.isLoggedIn);
   return (
     <Route {...rest}>
-      {auth.isLoggedIn ? children : <Redirect to="/login" />}
+      {auth.isLoggedIn === true ? children : <Redirect to="/login" />}
```

Comment:
I'm assuming this change is because the deleted line wasn't working? or does === force it to be true, whereas just auth.isLoggedIn can be only truthy or even falsey?